src/components/seeds/database/EditColumns.tsx

```diff
@@ -217,4 +227,23 @@ function sections(): Section[] {
       ],
     },
   ];
+
+  if (weightUnitsEnabled) {
```

Comment:
Can you add a note: when we undo the feature check - we should move this code to insert columns in the right place.. without the splice.